HealthcareSystem/GraphicalEditor/Models/MapObjectRelated/MapObjectDoor.cs

```diff
@@ -13,6 +14,7 @@ namespace GraphicalEditor.Models.MapObjectRelated
 {
     public class MapObjectDoor
     {
+        [JsonIgnore]
         public Rectangle Rectangle { get; set; }
         public MapObjectMetrics MapObjectMetrics { get; set; }
         public MapObjectDoorOrientation MapObjectDoorOrientation { get; set; }
```

Comment:
Why we are using this 'JsonIgnore' here?

Reply:
We only need metrics and orientation in json file. Rectangle has too many properties that will only unnecessary extend the file.

Reply:
Thanks for the explanation.